src/EnergyPlus/DXCoils.cc

```diff
@@ -3005,6 +3005,7 @@ void GetDXCoils(EnergyPlusData &state)
             ++DXCoilNum;
             auto const &fields = instance.value();
             std::string const &thisObjectName = instance.key();
+            s_ip->markObjectAsUsed(CurrentModuleObject, thisObjectName);
```

Comment:
Yes. If getObjectItem is removed, which marks the object as used, then that functionality must be restored in json processing.